cambl-test.lisp

```diff
@@ -122,13 +127,13 @@
 	(assert-value-equal x12 x13)
 	(assert-equal 2 (display-precision x13))))
 
-    (assert-equal "$2.000,00"
+    (assert-equal "$2,000.00"
```

Comment:
Why are you changing this?  The point of the test is that `$2.000,00` is parsed correctly, by observing the use of European decimal markers.

Reply:
The ```print-value-to-string``` function always formats amounts using the American markers. So I changed the tests accordingly (checking that $2000 $2,000.00 and $2.000,00 are all printed as $2,000.00).

Apparently in the past there was a ```*european-style*``` parameter that got removed, and I thought that these tests expecting amounts formatted using European style were left by mistake in commit 08c9aa7f64e4990f5819725c9dd8852f82cd2d1f.

Reply:
That's odd, I wonder why that was removed... OK, then I suppose there's no problem with these changes.